gitlabbot/gitlabbot/db.go

```diff
@@ -112,6 +112,28 @@ func (d *DB) GetSubscriptionForRepoExists(convID chat1.ConvIDStr, repo string) (
 	}
 }
 
+func (d *DB) GetAllSubscriptionsForConvID(convID chat1.ConvIDStr) (repos []string, branches []string, err error) {
```

Comment:
can you just make a 
```
type subscription struct { 
    repo string; 
    branch string
}
``` 
or maybe
```
type subscription struct { 
    repo string; 
    branches []string
}
```

to hold the response instead of separate arrays?